.github/workflows/release.yaml

```diff
@@ -0,0 +1,59 @@
+# This action assumes that there is a REL-commit which already has a
+# Markdown-formatted git tag. Hence the CHANGELOG is already adjusted
+# and it's decided what should be in the release.
+# This action only ensures the release is done with the proper contents
+# and that it's announced with a Github release.
+name: Publish Python Package to PyPI
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  build_and_publish:
+    # this doesn't make sense if you don't have the PyPI secret
+    if: github.repository == 'py-pdf/pypdf'
+    name: Publish a new version of pypdf
+    runs-on: ubuntu-latest
+
+    steps:
+      # Ensure it's on PyPI
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+
+      - name: Install Flit
+        run: |
+          python -m pip install --upgrade pip
+          pip install flit
+
+      - name: Publish Package to PyPI🚀
+        env:
+          FLIT_USERNAME: '__token__'
+          FLIT_PASSWORD: ${{ secrets.FLIT_PASSWORD }}
+        run: |
+          flit publish
+
+      # Create the Github Page
+      - name: Prepare variables
+        id: prepare_variables
+        run: |
+          git fetch --tags --force
+          latest_tag=$(git describe --tags --abbrev=0)
```

Comment:
This value is already available under the `${GITHUB_REF_NAME}` environment variable (https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) as well as the special `${{ github.ref_name }}` build context (https://docs.github.com/en/actions/learn-github-actions/contexts#github-context).